src/components/Instructions/GettingStarted/GettingStarted.js

```diff
@@ -874,6 +874,30 @@ const GettingStarted = ({ config }) => (
                     />
                   </a>
 
+                  <br />
+                  The following fields are optional and can be used for
+                  more advanced configuration:
+
+                  <ul>
+                    <li>
+                      <code>hidden</code>
+                      : if set to
+                      {' '}
+                      <code>true</code>
+                      , the publication will not
+                      be displayed.
+                    </li>
+                    <li>
+                      <code>collapsed</code>
+                      : if set to
+                      {' '}
+                      <code>true</code>
+                      , the publication will link
+                      to a separate page containing a list of sections instead
+                      of listing each section on the home page.
+                    </li>
+                  </ul>
```

Comment:
I wasn't sure the best way to include these fields in the Getting Started page, or even if they should be included. Displaying them below the other options and marking them as advanced seemed like a good way to do it.

---

<img width="931" alt="sn-3" src="https://user-images.githubusercontent.com/3039310/104341408-c37e6e00-54c7-11eb-9bae-063d21a8a423.png">